### PR TITLE
Markdown link in using-auto-tls.md is outdated

### PR DESCRIPTION
**Pull request by MIBc**
## Proposed Changes 
* Update install link for gloo and contour.
* Update ClusterIssuer install link.

## Fixes 
JUnit test error in `docs/serving/using-auto-tls.md`. Because of outdated  markdown link.

### CODE REVIEW

File: docs/serving/using-auto-tls.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ You must meet the following prerequisites to enable auto TLS:
 - The following must be installed on your Knative cluter:
   - [Knative Serving](../install/).
   - [Istio with SDS, version 1.3 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway),
-    [Contour, version 1.1 or higher](../install/Knative-with-Contour.md),
-    or [Gloo, version 0.18.16 or higher](../install/Knative-with-Gloo.md).
+    [Contour, version 1.1 or higher](https://github.com/knative/docs/blob/master/docs/install/knative-with-any-k8s.md#installing-the-serving-component),
```
Comment (evankanderson) [inline docs/serving/using-auto-tls.md]: I think here you want to link to `../install/any-kubernetes-cluster.md#installing-the-serving-component`.

Comment (MIBc) [reply docs/serving/using-auto-tls.md]: Yes. Thanks. I have fixed it.

**Pull request merged**